test/permit_docker.bats

```diff
@@ -58,5 +58,5 @@ teardown() {
   assert_output --partial "250 2.0.0 Ok: queued as "
 
   repeat_until_success_or_timeout 60 run docker exec mail_smtponly_second_network /bin/sh -c 'grep -cE "to=<user2\@external.tld>.*status\=sent" /var/log/mail/mail.log'
-  [ "${status}" -ge 0 ]
+  [[ "${status}" -ge 0 ]]
```

Comment:
I mentioned this file with the same quote question raised on `test/mail_smtponly.bats`, probably want to apply that here too:

```suggestion
  [[ ${status} -ge 0 ]]
```

Reply:
Yes :) I guess there are some other instances - we can just "bake them in whilst working on the project" when we encounter them.